src/log.py

```diff
@@ -0,0 +1,55 @@
+"""
+log
+
+Author: Norma Chang
+"""
+import json
+import os.path
+import time
+
+
+class Log:
+    """
+    Logging class.
+
+    Logs data from vehicle.
+    """
+    def __init__(self, name):
+        self.file_name = ""
+        self._start_time = time.strftime('%b %d %Y %H:%M:%S', time.gmtime())
+        self._start_time = self._start_time.replace(" ", "").replace(":", "")
+
+        self.file_name = name + self._start_time + '.log'
+
+
+    def write_to_file(self, dictionary):
+        """
+        Write dictionary into text file.
+
+        :param: dictionary: The dictionary passed in from a class. 
+        :type: dict
```

Comment:
Fix this one too.

Reply:
im confused about what you would like me to change here. Im fairly certain dict is the dictionary type.